src/internal-charts/internalLine.js

```diff
@@ -172,34 +178,64 @@ const InternalLine = {
 
   /**
   @private
-  @function
+  @function updateColors
   @description Updates color of line on chart after initial render
   @param {Array} colors Array of colors to update the chart to
   @param {Object} context Chart object
   */
-  updateColors(colors, context) {
+  updateColors(context) {
     context.element.select('svg')
-        .select('.line')
+        .selectAll('.line')
         .style({
           fill: 'none',
-          stroke: context.getColors[0],
           'stroke-width': 'crispEdges',
+        })
+        .style('stroke', (d, i) => {
+          return context.getColors[i];
         });
 
     return context;
   },
 
+  /**
+  @private
+  @function convertData
+  @description Converts the data needed for the chart
+  @param {Object} context Chart object
+  */
   convertData(context) {
-    context.data = utils.parseTimeData(context.data, context.xColumnName);
-    context.data = utils.parseNumberData(context.data, context.yColumnName);
+    context.data = utils.parseTimeData(context.data, context.xColumnName, '%Y%m%d');
```

Comment:
I have to add a user function that will let them use custom time formats. Right now it's hard coded for multilineData.csv.